addons/hw_drivers/tools/helpers.py

```diff
@@ -262,12 +294,17 @@ def get_ssid():
     process_grep = subprocess.Popen(['grep', 'ESSID:"'], stdin=process_iwconfig.stdout, stdout=subprocess.PIPE)
     return subprocess.check_output(['sed', 's/.*"\\(.*\\)"/\\1/'], stdin=process_grep.stdout).decode('utf-8').rstrip()
 
+
+@cache
```

Comment:
This has broken connecting to DB because the empty server URL gets cached. We could either revert it or add `get_odoo_server_url.cache_clear()` in the `/server_connect` and `/server_clear` routes

Reply:
@MaxTheWhale maybe you were biased as if you try in SSH by running odoo manually it does not restart in the terminal ?